### PR TITLE
Added CORS support

### PR DESCRIPTION
**Pull request by not-fl3**
Added "--cors" - Enable CORS via the Access-Control-Allow-Origin header. Inspired by the same flag of node's "https-server".

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -183,6 +188,7 @@ fn main() {
     let range = !matches.is_present("norange");
     let cert = matches.value_of("cert");
     let certpass = matches.value_of("certpass");
+    let cors = matches.occurrences_of("cors") != 0;
```
Comment (TheWaWaR) [inline src/main.rs]: Maybe `matches.is_present("")` ?

File: src/main.rs [hunk 2]
```diff
@@ -230,6 +236,7 @@ Address: {}
             enable_string(index),
             enable_string(upload),
             enable_string(cache),
+            (if cors { "enabled" } else { "disabled" }).to_string(),
```
Comment (TheWaWaR) [inline src/main.rs]: Maybe `enable_string(cors),` ?

Comment (not-fl3) [reply src/main.rs]: My bad, fixed.

File: src/main.rs (hunk 1 above)
Comment (not-fl3) [reply src/main.rs]: sure

**Pull request merged**